src/XrdHttp/XrdHttpExtHandler.hh

```diff
@@ -75,6 +75,12 @@ public:
   /// Sends a basic response. If the length is < 0 then it is calculated internally
   int SendSimpleResp(int code, const char *desc, const char *header_to_add, const char *body, long long bodylen);
 
+  // Start response to the client; often followed by data in multiple chunks
+  int StartSimpleResp(int code, const char *desc, const char *header_to_add, long long bodylen, bool keepalive);
```

Comment:
@amadio Wasn't the ABI CI supposed to break here?

Reply:
No, because `XrdHttp` has no `SONAME`. However, this change does break ExtHandler plugins, so I may have to adjust the build to include more libraries.